cmd/buildtools/k0s.go

```diff
@@ -0,0 +1,250 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	apkIndexURL = "https://packages.wolfi.dev/os/x86_64/APKINDEX.tar.gz"
```

Comment:
https://packages.wolfi.dev/os/x86_64/APKINDEX.json exists and may be much friendlier to parse and you dont have to extract it. Since you've already completed the work take this comment or leave it the way it is.

Reply:
for some reason there are versions that don't exist in that json file. for example, `coredns` version `1.11.3` exists in the tgz, but not in the json file.

Reply:
really weird